recurring_contract/wizards/recurring_invoicer_wizard.py

```diff
@@ -25,12 +37,16 @@ class InvoicerWizard(models.TransientModel):
 
     @api.multi
     def generate(self):
-        date_limit = datetime.date.today() + relativedelta(months=+1)
+
+        @memoize
```

Comment:
Does this mean that if we don't restart the server it will always return the same limit date (because date.today() is no more computed) ?

Reply:
You're right I hadn't thought about it. We could pass the day as well so it would only be computed once per day but it seems at this point like an unnecessary optimization.

Reply:
So I think it's best to remove the optimization. I didn't know about this @memoize decorator, it could be useful for other things but here I think it's not.